common/components/Header/components/GasPriceDropdown.jsx

```diff
@@ -0,0 +1,100 @@
+// @flow
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import './GasPriceDropdown.scss';
+
+export default class GasPriceDropdown extends Component {
+  constructor(props) {
```

Comment:
Looks like i am the only one, but i think code w/o constructors (where they not needed) are easier to read.

Here you can delete whole consturctor thing and replace it with 
`
export default class GasPriceDropdown extends Component {
  state = {
   expanded: false
  }
`